samples/training/s3_sample_data.py

```diff
@@ -0,0 +1,21 @@
+import urllib.request
+from urllib.parse import urlparse
+
+###################################################################
+# This is the only thing that you need to change to run this code
+# Give the name of your S3 bucket
```

Comment:
nit: To make this a standalone script, this should probably have some comments about region and credentials. wdyt ?

Reply:
The script doesn't ask for their region only uploads to their s3 bucket. Is it necessary to remind user to pass a bucket in the region they are currently using as default?

Reply:
ya, you could mention that.